### PR TITLE
add leanpkg.bat

### PR DESCRIPTION
**Pull request by emberian**

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -477,9 +478,15 @@ add_custom_target(clean-std-lib
 add_custom_target(clean-olean
   DEPENDS clean-std-lib)
 
-install(FILES "${CMAKE_SOURCE_DIR}/../bin/leanpkg"
-  DESTINATION bin
-  PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
+if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+  install(FILES "${CMAKE_SOURCE_DIR}/../bin/leanpkg.bat" "${CMAKE_SOURCE_DIR}/../bin/leanpkg.bat"
```
Comment (gebner) [inline src/CMakeLists.txt]: This installs only the `.bat` file and not the shell script on windows.  Does the `.bat` file work in the msys2 shell as well?

Comment (emberian) [reply src/CMakeLists.txt]: I just cargo culted the "leanemacs.bat" lines below.

I don't use the msys2 shell though, and the ZIP I download from github doesn't have the shell scripts.

Comment (emberian) [reply src/CMakeLists.txt]: (I'm still confused about the leanemacs.bat line, I'm pretty sure it's a mistake to specify the same path twice)

Comment (gebner) [reply src/CMakeLists.txt]: I don't think anybody actively tests the `.bat` files.  If you find problems with them, please fix (or report) them.

You can generate the release zip by running `make package` if you want to test it.

Comment (leodemoura) [reply src/CMakeLists.txt]: @gebner @cmr Should we close this PR?

Comment (emberian) [reply src/CMakeLists.txt]: I will fix it.

**Comment by emberian** [comment]
@gebner this is ready for another review (it also combines that other PR, which had some changes leaked into it by accident))

**Comment by gebner** [comment]
Thanks, all issues have been resolved.

**Pull request merged**